vendor/k8s.io/kube-openapi/pkg/handler/handler.go

```diff
@@ -77,6 +78,10 @@ func computeETag(data []byte) string {
 	return fmt.Sprintf("\"%X\"", sha512.Sum512(data))
 }
 
+// NOTE: [DEPRECATION] We will announce deprecation for format-separated endpoints for OpenAPI spec,
```

Comment:
Switch is in 1.10

Reply:
Sent out PR to correct that in kube-openapi